extensions/wikia/ARecoveryEngine/js/adBlockDetection.js

```diff
@@ -1,38 +1,32 @@
 /*global define*/
-define('ext.wikia.aRecoveryEngine.recovery.sourcePoint', [
+define('ext.wikia.aRecoveryEngine.adBlockDetection', [
```

Comment:
IMO this module has too many responsibilities. I'd move `isEnabled`, `getSafeUri`, `verifyContent`, `track` to another, second abstraction - "recovery". Then AdEngineRunner can really depnend on recovery (which depends on detection) instead of detection directly.